apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -97,8 +98,16 @@ def create(
             All named columns must exist in the schema, and at least one
             index column name is required.
 
-        :param platform_config: Platform-specific options used to create this DataFrame,
-            provided via ``{"tiledb": {"create": ...}}`` nested keys.
+        :param domains: An optional sequence of min-max tuples specifying the
+            domain of each index column, including room for any intended future
+            appends.  If provided, this sequence must have the same length as
+            `index_column_names`, and the index-column domains will be as specified.
+            If omitted entirely, or if ``None`` in a given dimension, the
+            corresponding index-column domain will use the maximum possible values.
```

Comment:
max possible values - what does this mean?  Is it "all possible values for the column type"?  (I believe this is the intent)

Reply:
updated